Source/RageCommunity.Library/Extensions/VectorExtensions.cs

```diff
@@ -116,5 +116,26 @@ public static bool IsOnScreen(this Vector3 position)
         {
             return NativeWrappers.GetHudScreenPositionFromWorldPosition(position, out float _, out float _);
         }
+        /// <summary>
+        /// Gets the nearest safe position to place a <see cref="Ped"/>
+        /// </summary>
+        /// <param name="positon">The position to check around</param>
+        /// <returns>If successfull, returns the <see cref="Vector3"/> result, otherwise returns <see cref="Vector3.Zero"/></returns>
```

Comment:
```suggestion
        /// <returns>If successful, returns the <see cref="Vector3"/> result, otherwise returns <see cref="Vector3.Zero"/></returns>
```